src/brevitas_examples/common/generative/quant_blocks.py

```diff
@@ -16,79 +16,6 @@
 from brevitas.function.ops_ste import abs_binary_sign_grad
 
 
-class OverSubChannelBlockView(brevitas.jit.ScriptModule):
```

Comment:
Are any of these classes used over at `optimum-amd`?

Reply:
I suspect not. I see you've changed to definition of the quantizers in common/generative to use the new features, so I expect this will work. We should test it though (not for this PR)

Reply:
I think we should be good but at some point the quantizers will move as well..